data/json/npcs/missiondef.json

```diff
@@ -1347,7 +1347,7 @@
     "followup": "MISSION_RANCH_FOREMAN_10",
     "dialogue": {
       "describe": "We need help...",
-      "offer": "Disease and infection remains a persistent problem among the refugees.  Without dedicated medical personnel and facilities I doubt everyone will be willing to stick around when the next outbreak happens.  Until we can get a former medic or nurse I'm just going to have to improvise.  Sterilization would be the first step I imagine.  Bring me five gallons of bleach so we can get started.",
+      "offer": "Disease and infection remains a persistent problem among the refugees.  Without dedicated medical personnel and facilities I doubt everyone will be willing to stick around when the next outbreak happens.  Until we can get a former medic or nurse I'm just going to have to improvise.  Sterilization would be the first step I imagine.  Bring me 150 units of bleach so we can get started.",
```

Comment:
"150 units" doesn't sound like a human speech. "N bottles of bleach" will be much better.

Reply:
The point of this change was because “5 gallons of bleach” was likely to be wrong.
Note that many other speeches include the word “unit” for some reason.
If you can suggest a change which is both correct AND not weird, we can talk.
Ideally I would use litres. How do “bleach units” convert to litres, anyway?